config/SkyBotEphemerisQuery.py

```diff
@@ -0,0 +1,4 @@
+# DECam-specific config for lsst.ap.association.skyBotEphemerisQuery.SkyBotEphemerisQueryTask
+
+config.observerCode = "807"        # IAU code for CTIO
```

Comment:
Do we drop the telescope location somewhere in the data pipeline? We know exactly where it is at the start of the processing (ObservationInfo calculates it) and it's in VisitInfo. I'm assuming then that this code is using catalogs and the catalogs don't have sufficient metadata because we haven't propagated some header?

Reply:
No, it's not a matter of header propagation, as this task is supposed to be run _before_ processing. The observer code is for querying the Skybot server. It [does allow coordinate input instead](https://ssp.imcce.fr/webservices/skybot/howto/conesearch/#howto-observer), but I don't know how practical that would be to switch to that, given that the use of `VisitInfo` as task input is supposed to be a stopgap.